sentry/src/main/java/io/sentry/transport/AsyncHttpTransport.java

```diff
@@ -224,6 +246,14 @@ public void run() {
 
             options.getLogger().log(SentryLevel.ERROR, message);
 
+            if (!(sentrySdkHint instanceof Retryable)) {
+              // ignore e.g. 429 as we're not the ones actively dropping
+              if (result.getResponseCode() >= 500) {
```

Comment:
I guess I've made this question in the other transport class, that makes sense.
Did we confirm with them (ingest) that we should report when its >= 500?

Reply:
My assumption was 4xx errors would go into the SDK bug category which we don't want to track using client reports (for now).

Reply:
Yes, we should not report 4xx since it's client-side or rate-limiting, 5xx means the server has an error, do they recover and report client reports or not? that's the open question.

Reply:
Asked on client infra channel, outcome here: https://github.com/getsentry/develop/pull/554
> SDKs can put a simple optional check for HTTP status codes in place where any code `>= 400` except `429` will be recorded as `network_error`.